leapp/cli/upgrade/__init__.py

```diff
@@ -183,6 +183,18 @@ def process_whitelist_experimental(repositories, workflow, configuration, logger
             raise CommandError(msg)
 
 
+def process_report_schema(args, cfg):
+    report_schema = None
+    default_report_schema = cfg.get('report', 'schema')
+    if args.report_schema and args.report_schema != default_report_schema:
+        if args.report_schema < default_report_schema:
+            report_schema = args.report_schema
+        else:
+            raise CommandError('--report-schema version can not be greater that the '
+                               'actual {} one.'.format(default_report_schema))
+    return report_schema
```

Comment:
Just to be sure, is it intended, by default to return `None` if args.report_schema isn't set or equal to the default_report_schema?

Reply:
Yeah, it was a shortcut to not doing any checks later if report messages processing needs to be done at all (otherwise you won't differentiate between 'convert to 1.1.0' and 'don't do anything, because 1.1.0 is the latest version anyway' without comparing the passed version to the actual from the config file).
But now I think I know a way to improve it, thank you!

Reply:
Thinking about that, what about to add the `default` value argument for `add_option`?